File: Server/src/GHR.Domain/DataBase/Funcionarios/DadoPessoal.cs
using System;
using GHR.Domain.DataBase.Empresas;

namespace GHR.Domain.DataBase.Funcionarios
{
    public class DadoPessoal
    {
        public int Id { get; set; } 
        public string CPF { get; set; }
        public string TituloEleitor { get; set; }  
        public string Identidade { get; set; }
        public DateTime DataExpedicaoIdentidade { get; set; }     
        public string OrgaoExpedicaoIdentidade { get; set; }
        public string UfIdentidade { get; set; }
        public string EstadoCivil { get; set; } 
        public string CarteiraTrabalho { get; set; }
        public DateTime DataExpedicaoCarteiraTrabalho { get; set; } 
    }
}

File: SPA/GHR-App/src/app/models/funcionarios/DadoPessoal.ts
import { DateTimeFormatPipe } from './../../helpers/DateTimeFormat.pipe';
export class DadoPessoal {
  id: number;
  cpf: string;
  tituloEleitor: string;
  identidade: string;
  dataExpedicaoIdentidade: Date;
  orgaoExpedicaoIdentidade: Date
  UfIdentidade: string;
  estadoCivil: string;
  carteiraTrabalho: string;
  dataExpedicaoCarteiraTrabalho: string;
}
